pom.xml

```diff
@@ -1183,6 +1191,13 @@
                         <exclude>org/apache/druid/benchmark/**/*</exclude>  <!-- benchmarks -->
                         <exclude>org/apache/druid/**/*Benchmark*</exclude>  <!-- benchmarks -->
                         <exclude>org/apache/druid/testing/**/*</exclude>  <!-- integration-tests -->
+                        <!-- The ITs has test code sprinked through the module tree. Remove the following
+                             once the old version is retired. -->
```

Comment:
Can we create an issue for this if we haven't already? This will help us keep track of the todo's